website/static/js/projectorganizer.js

```diff
@@ -1029,10 +1039,10 @@ function canAcceptDrop(items, folder) {
     if (hasFolders) {
         canDrop = canDrop && folder.data.permissions.acceptsFolders;
     }
-    if (copyMode === 'move') {
+    if (theCopyMode === 'move') {
```

Comment:
So, there's a global variable the project organizer called copyMode, which is how the Project Organizer handles copy mode. But I needed a way to optionally pass it in, and this seemed the best way. Do you have an alternate suggestion? I could refactor copyMode so that it's globalCopyMode and change the local version to copyMode.